server/src/test/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationLifecycleTest.java

```diff
@@ -29,12 +40,26 @@
 
     @Before
     public void setUp() throws Exception {
-        PluginFileAccess access = new PluginFileAccess(new File(""), tmpdir, mock(Log.class));
-        EffectiveModel effectiveModel = new EffectiveModelFactory().create(TestCsars.VALID_MINIMAL_DOCKER_TEMPLATE, logMock());
-
+        PluginFileAccess accessL = new PluginFileAccess(
+            new File(tmpdir, "sourceDir"),
+            new File(tmpdir, "targetDir"),
+            mock(Log.class));
```

Comment:
you could simply use `logMock()` instead of `mock(Log.class)`. The calls to getLogger() are already stubbed.